.github/workflows/amoy_deb_profiles.yml

```diff
@@ -0,0 +1,387 @@
+name: amoy_deb_profiles
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
We need to change to master in bor as well.

Reply:
Updated